tests/testthat/mainTests/ethnicity_characteristics_filter_wrong_values.csv

```diff
@@ -0,0 +1,958 @@
+"time_period","time_identifier","geographic_level","country_code","country_name","region_code","region_name","characteristic_group","characteristic","full_time_equivalent","headcount"
```

Comment:
This file is missing a .meta file?

Reply:
As above, added in and expanded unit tests to cover.